websites/O/OG Network/metadata.json

```
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "630070645874622494",
+		"name": "johnfries"
+	},
+	"service": "OG Network",
+	"description": {
+		"en": "OG Network website presence"
+	},
+	"url": [
+		"og-network.net",
+		"shop-og-network.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://www.og-network.net/assets/logo.png",
```

Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

```suggestion
	"logo": "https://i.imgur.com/6jC2EYO.png",
```